Update Project is updated by using given feedbacks

ozovalihasan opened the pull request:
Lastly, TSE gave feedback ( https://github.com/ozovalihasan/youtubeCloneByHasan/pull/2#pullrequestreview-382038703 ). 
As a result, the following changes are made.
-http://archive.vn/Bss88 is used as a source.
-The navigation bar is arranged.
-Padding of video and sidebar are arranged.
-Part of video information is arranged
-More suggested video are added into sidebar
-Footer is added.
-index.html and styles.css are checked by using linter
-All images are moved into one folder.
-Live demo link is changed, but mainParts branch is used for live demo; it will be changed after mainParts branch is merged.
-I continued to work alone.

ozovalihasan committed:
Add All files of https://github.com/ozovalihasan/youtubeCloneByHasan.
TSE of Microverse gave a feedback about branches. Now, "master" branch have only README.md file, and other files are added as a part of new branch.

ozovalihasan committed:
Add .gitignore file is added.

ozovalihasan committed:
Fix CSS file fixed.
Errors found using CSS-linters (https://github.com/microverseinc/linters-config/tree/master/css)

ozovalihasan committed:
Fix CSS file fixed.
Errors found using CSS-linters (https://github.com/microverseinc/linters-config/tree/master/css) are fixed.

ozovalihasan committed:
Add .stickler.yml and stylelint.config.js are added.
To set-up stickler, these files are needed

ozovalihasan committed:
Fix html element of index.html is fixed
There is a warning as a result of https://validator.w3.org/nu/#file. "lang" attribute was missed, it is added

ozovalihasan committed:
Fix Like bar button is fixed.
There is error shwon by https://validator.w3.org/ about button. So, like bar button fixed as having 5 separate button

ozovalihasan committed:
Update Margin are added to left and right of video.

TSE gave a feedback about margin of left and right of video.

ozovalihasan committed:
Document style.css file is moved into CSS folder created
To reset page it is necessary to create another css file. So, all css file will be in the same folder.

ozovalihasan committed:
Add reset.css file is created
To reset browser settings reset.css is needed (https://speckyboy.com/good-bad-css-practices/)

ozovalihasan committed:
Fix Position of navigation bar is fixed.
After using reset.css, it is necessary to fix position of navigation bar

ozovalihasan committed:
Fix Position of video and sidebar is fixed
Navigation bar was on top of video and sidebar.

ozovalihasan committed:
Update Screenshot of cloned Youtube page is updated

ozovalihasan committed:
Fix reset.css file fixed
reset.css file is checked by https://github.com/microverseinc/linters-config. Errors are fixed.

ozovalihasan committed:
Document All image files moved into a file.
Images aren't seen very well in root file. They are moved into "images" folder

ozovalihasan committed:
Update Project is updated by help of feedback of TSE

Padding are added to left and right of video and sidebar
Navigation bar and information of video are arranged by using http://archive.vn/Bss88 as a source
Added more suggested video into sidebar
Footer is added

ozovalihasan committed:
Update Position of icons in nav bar is arranged

ozovalihasan committed:
Fix Errors shown by html and css linters are fixed

ozovalihasan committed:
Update README file is updated.

Title of project is changed.
Live demo link is changed, but it is directed to mainParts branch because master branch doesn't have index.html file.

uryelah commented on index.html:
```diff
@@ -0,0 +1,214 @@
+            -->
+
+            <!-- 
+            <div id="camera">
```
- [x] Please remove commented out code form your files, if you still want to reuse it in the future you can save it in another branch

uryelah commented on index.html:
```diff
@@ -0,0 +1,214 @@
+            <aside id="sidebar">
+                <div id="nextBar">
+                    <h1 id="nextText">Up Next</h1>
+                    <h1 id="autoplay">Autoplay</h1>
```
- [x] The autoplay is a description of what the toggle button does, not a title. Avoid using h tags for elements that don't function as titles.

uryelah commented on index.html:
```diff
@@ -0,0 +1,214 @@
+                        <div class="sugProperites">
+                            <h2 class="sugName">Contributing to Open Source Part I: The Easy Way - YouTube</h2>
+                            <h2 class="sugAuthor">The Odin Project</h2>
+                            <h2 class="sugView">33 K view</h2>
```
- [x] Again, do not overuse h tags. [Read more on how to use them here.](https://www.w3schools.com/tags/tag_hn.asp)
In the case of the recommended video only the video title would make semantic sense as an h2.

uryelah commented on index.html:
```diff
@@ -0,0 +1,214 @@
+                    <div class="suggestionElement"> 
+                        <img class="sugImage" src="sidebar/suggestion3.png" alt="One of suggestion videos" >
+                        <div class="sugProperites">
+                            <h2 class="sugName">Anjana Vakil: Javascript ile Fonksiyonel Programlamayı ... </h2>
```
- [x] Try embedding the suggestion video thumbnail and title inside `<a>` tags. In the original Youtuber page they link you to another video, when you clone a page keep in mind to also mimic functionality whenever possible.

uryelah commented on index.html:
```diff
@@ -0,0 +1,214 @@
+        </main> 
+
+        <footer>
+           <img src="images/footer.png" alt="">
```
- [x] [IMPORTANT!] You should never just take a screenshot of a whole page are and use it as an actual element. 
Imagine how frustrating it would be for a user to try and select any text from the footer or to click one of its links.

uryelah commented on index.html:
```diff
@@ -0,0 +1,214 @@
+        
+        <main>
+            <br>
+            <br>
```
- [x] Don't use `<br>` tags. Positioning should be handled by CSS. [Read more here.](https://webmasters.stackexchange.com/questions/79199/why-isnt-the-html-break-tag-acceptable-anymore-for-top-and-bottom-margins)

uryelah commented on CSS/styles.css:
```diff
@@ -0,0 +1,286 @@
+  font-family: sans-serif;
+}
+
+#navbar {
```
- [x] Don't use ids to style. Use classes. [Read more here ](https://dev.to/clairecodes/reasons-not-to-use-ids-in-css-4ni4)on reasons why.

uryelah reviewed (changes requested):
Hello **Hasan Özovalı**, I'm Sarah, your reviewer for this PR 👩‍🚀  

**First of all welcome to Microverse, good job starting with the first project.🎇**
I think that as the last TSE mentioned you might need a refresher on best HTML and CSS practices, make sure to read the link he left, so if some of the required changes bellow are not very clear make sure to ask on Slack or in the comments of this PR about them.

There are still some issues to work through so hang in there 💪 

---------    

### Required changes 🛠 
---------

- [x] After receiving a code review don't make a new pull request, submit the same pull request for the next review. Remember this for this and the next projects.

- [x] In the navbar and under the video use actual icons, you could get them [in a site like FontAwesome](https://fontawesome.com/), do not take screenshots of the original icons on the page. When you take a screenshot you get an image with a background that can have a different color, without any transparency and possibly in a lower resolution.

- [x] The navbar 'signin' button should be an `<button>` with the correct styling and content, not a screenshot. (For the user to know an element is an actual button, make sure to change the `cursor` property to `pointer`).

- [x] The search bar button should be styled like the original and have an icon inside it, not a screenshot. You should also change the background color of the button yourself if needed. Try using your browser color picker,[ like the one Firefox has.](https://developer.mozilla.org/en-US/docs/Tools/Eyedropper)
 (For the user to know an element is an actual button, make sure to change the `cursor` property to `pointer`).

- [x] Make sure your navbar takes 100% of the page width.

- [x] For the subscribe button, user image and icons under the video. Do not use screenshots.
- Make the subscribe button by styling a button element, use the background, color, border, cursor, proprieties.
- Make the user image by embedding an image and change it's `border-radius`
- Make the buttons with icons as children, not screenshot images.
![video_like_requirement](https://user-images.githubusercontent.com/16193212/77753600-9b8d4f00-7008-11ea-9223-19e8bd209722.png)
![icons_requirement](https://user-images.githubusercontent.com/16193212/77753611-9e883f80-7008-11ea-98c0-8998edecf4f1.png)

- [x] The footer is not required, but if you still want to add it make sure to actually build it with HTML tags and CSS styling, never user screenshots to make a page element look like the real thing.
When you clone a page in the Microverse curriculum it should always be made with the correct tags and markup, just looking the same as a screenshot is not enough. 

- [x] The 'Up next' aside autoplay image could be made either with divs or by getting an image with a transparent background of a 'toggle' and placing it making sure it's not distorted. Do not use a screenshot, notice how it looks low resolution and with a different background color.
[Check the styling of the toggle here if you want to make it yourself.](https://www.w3schools.com/howto/howto_css_switch.asp)

- [x] For each 'Up next' video use the actual thumbnail images for the videos, not screenshots. 
You can easily get an image from a youtube thumbnail by placing your mouse above it, clicking the right button of your mouse and then clicking view image and saving the image.

uryelah commented:
**Kindly fix the REQUIRED issues and send another code review request.**

The changes might look like a lot but if you get used to following these practices the next reviews will surely be a breeze!

Cheers and Happy coding!👏👏👏

Code review by **Sarah Uryelah Silva Chamorro**

(Feel free to contact me on slack by the handle of  **sarah** if you have any additional question)

ozovalihasan committed:
Update The project has been updated considering given feedbacks.

All printscreens are deleted.
Icons provided by Font Awesome are added.
All buttons are arranged as button.
All buttons have pointer cursor property
All suggested videos are changed using related images.
All id elements are changed as class.
All h tags deleted, except they are used for titles.
a elements used for possible links.
Navigation bar is arranged.
Footer is added.
Toggle button is added.
Colors are found and used by using Colorpick Eyedropper.
html and css files are analyzed for possible errors. All found errors are fixed.

ozovalihasan committed:
Update Positions of like/unlike bar buttons are changed.

ozovalihasan commented:
Thanks for your feedback @uryelah. Thanks to your feedback, I learned a lot and updated my project.

ozovalihasan committed:
Fix All webp links are deleted. Added one icon nav. bar.

webp links are seen as deleted files because link of webp are changing in 24 hours.
So,all links of webp are deleted.
İcon of bell is added to navigation bar.
User icon is arranged as round.

ozovalihasan committed:
Update Color of bell is changed

SofiBretz commented:
### Changes required ♻️
### Hello @ozovalihasan! 🌟

Welcome to Microverse!

Great work on this project, but you need to fix the issues below before moving on ⬇️

**- [ ] Are you working alone on this project? I can see that you commit everything, where is your partner? please contact me on Slack.**

- [x] The 'subscribe', 'add', and 'share' buttons have to be on the right side of that section.

![3](https://user-images.githubusercontent.com/42303392/77989585-6caefb80-72dc-11ea-8696-42092927fac4.png)

Original Page:
![3 1](https://user-images.githubusercontent.com/42303392/77989582-6ae53800-72dc-11ea-9756-063002847fb1.png)

- [x] The proportions of the elements on the 'navbar' section have to be as the original page, consider following the original page.

![4](https://user-images.githubusercontent.com/42303392/77989784-df1fdb80-72dc-11ea-986d-e3b51a87dd52.png)

Original Page:
![4 1](https://user-images.githubusercontent.com/42303392/77989782-de874500-72dc-11ea-93ff-d9572e85102a.png)

[OPTIONAL]

- [x] If you worked alone on this project, please add only your personal information on the readme file.

_**After making the required changes please submit a new code review request**_
Good luck! 🍀 You are almost there! 💪

SofiBretz.

ozovalihasan committed:
Fix Elements of nav bar and position of subscribe button are fixed

More navigation bar elements are added.
Position of subscribe button is fixed and description part is arranged.
Comments of copy-pasted code and this code are deleted and toggle button arranged.
Errors of html and css codes  are checked and fixed

ozovalihasan commented:
-The navigation bar and position of the subscribe button are arranged. 
-Code used for toggle button is deleted and added as fixed.
-I worked alone

ozovalihasan committed:
Update Screenshot of project is updated

Dandush03 commented:
# :red_circle: Status: Status Required  :red_circle:
### Hey @ozovalihasan  :wave: 
## Good attempts on this one :muscle: :+1: 

## Your project looks great! but is a very important thing you have to consider before submitting another code review :point_down: 

## Nav-Bar
- [x] The Nav-Bar left section - element aren't aligned to the left as original
- [x] Icons arn't the same

##### Required:
![Screenshot from 2020-03-31 09-03-01](https://user-images.githubusercontent.com/51087936/78029436-942eb400-732e-11ea-9cd7-273616cb1191.png)
##### Yours: 
![Screenshot from 2020-03-31 09-03-34](https://user-images.githubusercontent.com/51087936/78029442-97c23b00-732e-11ea-9a0d-15c9a48f1882.png)

## Kindly address all these changes and submit a new code review

### Happy coding:computer: 

#### Daniel Laloush

ozovalihasan commented:
Update The screenshot of my project is updated. 
I worked alone because I started this project, but then he withdrew. So, I am alone, now.

mikenath223 commented:
### Pull request currently under review by Michgolden Ukeje (TSE).
Kindly exercise patience 🐱

mikenath223 reviewed (approved):
### CONGRATULATIONS TEAM 👏👏👏

This project meets the requirements for approval.
Great job working with the previous reviews to get the project to this stage.🎉👨‍💻

**STATUS: APPROVED ✅**

However kindly take note of this:
Please don't open a new Pull request when a reviewer has commented on your project and asked you to make adjustments to the project. **Simply make the adjustment on the same branch and request another review with the same Pull request link.**

Don't forget to fill the project completion form👍
Good luck with the next project.

Welcome to Microverse!
HAPPY CODING👨‍💻

ozovalihasan committed:
Update Readme file is updated.

Link of live demo is changed.
